packages/rich-text/src/RichTextEditor.styles.ts

```diff
@@ -28,6 +28,10 @@ export const styles = {
         },
       },
     },
+    // We need to reset LIC style due to conflicts between PARAGRAPH styles
+    'ul > li > div': {
+      margin: 0,
+    },
```

Comment:
I can explain this better. I think it's how this slate-plugins lib works internally. So when we map `ELEMENT_LIC` to type `BLOCKS.PARAGRAPH`, this will actually use what `BLOCKS.PARAGRAPH` is using as a component. This paragraph component has some margin styles. The `component` property of `ELEMENT_LIC` is completely ignored.